### ISSUE TITLE
Fail to install

### ISSUE THREAD

**Issue by yordis** [looztra/asdf-terraform-docs#2]
Using  `asdf install terraform-docs 0.9.1` (notice that it doesn't have a `v` in the version:

```shell
❯ asdf install terraform-docs 0.9.1
Downloading [terraform-docs] from https://github.com/segmentio/terraform-docs/releases/download/0.9.1/terraform-docs-0.9.1-darwin-amd64 to /var/folders/vs/k_3wkr1s2d3cjx7qpgyj8l7w0000gn/T/asdf_XXXXXXXX.wVpisiIT/terraform-docs-0.9.1-darwin-amd64
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     30      0 --:--:-- --:--:-- --:--:--    30
Creating bin directory
Cleaning previous binaries
Copying binary
```

It writes to disk `Not Found` in the binary name.

You must use `v0.9.1` in order to make it work.

**Comment by WillBeebe** [comment]
Thanks for this! I was stuck too.

`asdf list all terraform-docs` returns nothing.

**Comment by bdellegrazie** [comment]
@looztra: I think you need a `-sL` instead of just `-s` on the curl command to retrieve the versions now due to Github API changes

**Comment by looztra** [comment]
thank you for reporting this, working on it

**Comment by looztra** [comment]
This has been fixed by #3 

run `asdf plugin update terraform-docs` to update your local copy of the plugin

@yordis you have to use `asdf install terraform-docs v0.9.1` so that it works. Actually, you have to use of the versions returned by `asdf list all terraform-docs` when you execute `asdf install terraform-docs XXX`.
This command (list all) was failing because the git repo coordinates changed (and the plugin didn't follow the redirect).
This is now fixed.
